docker_interop/app/src/main/java/tech/pegasys/qbft/interop/DockerComposeYaml.java

```diff
@@ -0,0 +1,90 @@
+package tech.pegasys.qbft.interop;
+
+import org.hyperledger.besu.crypto.KeyPair;
+import org.hyperledger.besu.crypto.SECP256K1;
+import org.yaml.snakeyaml.DumperOptions;
+import org.yaml.snakeyaml.Yaml;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Path;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class DockerComposeYaml {
+    static final int START_IP = 10;
+    static final int START_RPC_PORT = 8541;
+    static final int START_P2P_PORT = 35300;
+    static final String IP_PREFIX = "172.16.239.";
```

Comment:
How can be sure that the network will have this range? Is this defined somewhere as part of the docker network?

Reply:
that is correct. docker-compose-template.yml defines it.